testsuite/features/support/env.rb

```diff
@@ -319,6 +321,11 @@ def enable_assertions
   skip_this_scenario unless $long_tests_enabled
 end
 
+# do test only if we want to run service pack migration
+Before('@service_pack_migration') do
+  skip_this_scenario unless $service_pack_migration_enabled
+end
```

Comment:
As far as I see the test strategy for the SP migration, we want to execute it always.
As we will deploy from the terraform file a SLES15SP1 version, and we want to test SLES15SP2 version for all our test suites features. So, I'd say is not optional to migrate to SP2.

Reply:
Agree. Once merged, let's keep this tag for some CI runs (passing the env variable to the pipe in order to run the SP migration) for easy comparison with/without SP migration; if we don't observe breakages, then it has sense to drop this optional tag.